wiki/en/Server-Linux.md

```diff
@@ -29,9 +29,9 @@ You can control Jamulus with the `systemctl` command. For example, to stop the s
 
 `sudo systemctl stop jamulus-headless`
 
-### Running in public mode
+### Running in Registered mode
 
-The following minimum setup is required to [run a public server](Running-a-Server#server-types):
+The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
```

Comment:
> run a Registered Server

The example in lines 35 to 55 is to "run a Server registered with a Public Directory" - it makes no mention of registering with a Custom Directory